core/src/main/java/com/orange/ouds/core/component/OudsDivider.kt

```diff
@@ -0,0 +1,141 @@
+import com.orange.ouds.foundation.utilities.EnumPreviewParameterProvider
+import com.orange.ouds.foundation.utilities.UiModePreviews
+import com.orange.ouds.theme.tokens.OudsColorKeyToken
+
+//TODO Add DSM link when available
+// <a href="https://unified-design-system.orange.com/" class="external" target="_blank">OUDS Divider design guidelines</a>
+/**
+ * An OUDS horizontal divider.
+ *
+ * This composable renders a horizontal line that can be used to visually separate content.
+ * The color of the divider can be specified using the [OudsDivider.Color] enum, and the thickness is defined by the current theme's divider border width.
+ *
+ * @param modifier [Modifier] applied to the divider.
+ * @param color The color of the divider, chosen from the [OudsDivider.Color] enum. Defaults set to `OudsDivider.Color.Default`.
+ *
+ * @sample com.orange.ouds.core.component.samples.OudsHorizontalDividerSample
+ */
+@Composable
+fun OudsHorizontalDivider(
+    modifier: Modifier = Modifier,
+    color: OudsDivider.Color = OudsDivider.Color.Default
+) {
+    HorizontalDivider(modifier = modifier, color = color.value, thickness = OudsTheme.componentsTokens.divider.borderWidth.value)
+}
+
+//TODO Add DSM link when available
+// <a href="https://unified-design-system.orange.com/" class="external" target="_blank">OUDS Divider design guidelines</a>
+/**
+ * An OUDS vertical divider.
+ *
+ * This composable renders a vertical line that can be used to visually separate content.
+ * The color of the divider can be specified using the [OudsDivider.Color] enum, and the thickness is defined by the current theme's divider border width.
+ *
+ * @param modifier [Modifier] applied to the divider.
+ * @param color The color of the divider, chosen from the [OudsDivider.Color] enum. Defaults set to `OudsDivider.Color.Default`.
+ *
+ * @sample com.orange.ouds.core.component.samples.OudsVerticalDividerSample
+ */
+@Composable
+fun OudsVerticalDivider(
+    modifier: Modifier = Modifier,
+    color: OudsDivider.Color = OudsDivider.Color.Default
+) {
+    VerticalDivider(modifier = modifier, color = color.value, thickness = OudsTheme.componentsTokens.divider.borderWidth.value)
+}
+
+/**
+ * Contains classes to build an [com.orange.ouds.core.component.OudsHorizontalDivider] or an [com.orange.ouds.core.component.OudsVerticalDivider].
+ */
+object OudsDivider {
+
+    /**
+     * Represents the possible colors for an [OudsHorizontalDivider] or an [OudsVerticalDivider].
+     * Each color corresponds to a specific color key token from the design system.
+     */
+    enum class Color {
+        AlwaysBlack,
+        AlwaysOnBlack,
+        AlwaysOnWhite,
+        AlwaysWhite,
+        BrandPrimary,
+        Default,
+        Emphasized,
+        Muted,
+        OnBrandPrimary;
+
+        val value: androidx.compose.ui.graphics.Color
+            @Composable
+            get() {
+                return when (this) {
+                    AlwaysBlack -> OudsColorKeyToken.Always.Black
```

Comment:
This has no impact and we also did this for `OudsColoredBox`, but we should get the color using `OudsTheme.colorScheme` instead, because we decided that key tokens should only be used as parameters of the OudsXxxTokens files for components.